nipype/interfaces/freesurfer/petsurfer.py

```diff
@@ -0,0 +1,651 @@
+    )      
+
+    lat = traits.Bool(
+        argstr="--lat", desc="lateralize tissue types"
+    )       
+
+    no_tfe = traits.Bool(
+        argstr="--no-tfe", desc="do not correction for tissue fraction effect (with --psf 0 turns off PVC entirely)"
+    )        
+
+    segpvfres = traits.Float(
+        argstr="--segpvfres %f",
+        desc="set the tissue fraction resolution parameter (def is 0.5)"
+    )   
+
+    rbv = traits.Bool(
+        argstr="--rbv", desc="perform RBV PVC"
+    )         
+
+    rbv_res = traits.Float(
+        argstr="--rbv-res %f",
+        desc="voxsize : set RBV voxel resolution (good for when standard res takes too much memory)"
+    ) 
+
+    mg = traits.List(
+        argstr="--id %s...", desc="Manually specify segmentation ids"
+    )
+
+    mg_ref_cerebral_wm = traits.Bool(
+        argstr="--mg-ref-cerebral-wm", desc=" set MG RefIds to 2 and 41"
+    )       
+
+    mg_ref_lobes_wm = traits.Bool(
+        argstr="--mg-ref-lobes-wm", desc="set MG RefIds to those for lobes when using wm subseg"
+    )      
+
+    mgx = traits.Float(
+        argstr="--mgx %f",
+        desc="gmxthresh : GLM-based Mueller-Gaertner PVC, gmxthresh is min gm pvf bet 0 and 1"
+    ) 
+
+    km_ref = traits.List(
+        argstr="--km-ref %s...", desc="RefId1 RefId2 ... : compute reference TAC for KM as mean of given RefIds"
+    )
+
+    km_hb = traits.List(
+        argstr="--km-hb %s...", desc="RefId1 RefId2 ... : compute HiBinding TAC for KM as mean of given RefIds"
+    ) 
+
+    ss = traits.Tuple(
+        traits.Float,
+        traits.Float,
+        traits.Float,
+        argstr="--ss %f %f %f",
+        desc="bpc scale dcf : steady-state analysis spec blood plasma concentration, unit scale and decay correction factor. You must also spec --km-ref. Turns off rescaling"
+    ) 
+
+    X = traits.Bool(
+        argstr="--X", desc="save X matrix in matlab4 format as X.mat (it will be big)"
+    ) 
+
+    y = traits.Bool(
+        argstr="--y", desc="save y matrix in matlab4 format as y.mat"
+    )
+
+    beta = traits.Bool(
+        argstr="--beta", desc="save beta matrix in matlab4 format as beta.mat"
+    )
+    
+    X0 = traits.Bool(
+        argstr="--X0", desc="save X0 matrix in matlab4 format as X0.mat (it will be big)"
+    )
+
+    save_input = traits.Bool(
+        argstr="--save-input", desc="saves rescaled input as input.rescaled.nii.gz"
+    )
+
+    save_eres = traits.Bool(
+        argstr="--save-eres", desc="saves residual error"
+    )
+
+    save_yhat = traits.Bool(
+        argstr="--save-yhat", desc="save signal estimate (yhat) smoothed with the PSF"
+    )
+
+    save_yhat_with_noise = traits.Tuple(
+        traits.Int,
+        traits.Int,
+        argstr="--ss %i %i",
+        desc="seed nreps : saves yhat with noise, seed < 0 for TOD"
+    ) 
+    
+    traits.Bool(
+        argstr="--save-yhat-with-noise", desc="save signal estimate (yhat) with noise"
+    ) 
+
```

Comment:
The first trait doesn't seem to match the name. The second isn't assigned to a variable.